docs/source/serialization.rst

```diff
@@ -78,13 +105,15 @@ Example of quantized BERT model export:
     python convert_graph_to_onnx.py --framework <pt, tf> --model bert-base-cased --quantize bert-base-cased.onnx
 
 .. note::
-    Quantization support requires ONNX Runtime >= 1.4.0
+    (1) Quantization support requires ONNX Runtime >= 1.4.0
```

Comment:
Same as above.

The three notes stacked together could be put in just one with an enumerated list however, it's a matter of preferences (see [here](https://66137-155220641-gh.circle-artifacts.com/0/docs/_build/html/serialization.html) for current preview).